seatunnel-server/seatunnel-app/src/main/java/org/apache/seatunnel/app/service/impl/JobExecutorServiceImpl.java

```diff
@@ -90,20 +100,15 @@ public Result<Long> jobExecute(Integer userId, Long jobDefineId, JobExecParam ex
     }
 
     public String writeJobConfigIntoConfFile(String jobConfig, Long jobDefineId) {
-        String projectRoot = System.getProperty("user.dir");
-        String filePath =
-                projectRoot + File.separator + "profile" + File.separator + jobDefineId + ".conf";
+        String projectRoot = System.getProperty(USER_DIR);
+        String dirPath = projectRoot + File.separator + DIR_PROFILE;
+        String filePath = dirPath + File.separator + jobDefineId + DOT_CONF;
         try {
-            File file = new File(filePath);
-            if (!file.exists()) {
-                file.getParentFile().mkdirs();
-            }
 
-            FileWriter fileWriter = new FileWriter(file);
-            BufferedWriter bufferedWriter = new BufferedWriter(fileWriter);
+            Path path = Paths.get(dirPath);
+            Files.createDirectories(path);
 
-            bufferedWriter.write(jobConfig);
-            bufferedWriter.close();
+            Files.write(Paths.get(filePath), jobConfig.getBytes(), StandardOpenOption.APPEND);
```

Comment:
Previously, the configuration file was overwritten every time it was regenerated. How will StandardOpenOption.APPEND work in this context? Have you tested it?

Reply:
Hi @arshadmohammad , thanks for the your valuable feedback and input, yes you are right if the file is open it will bind in the last of the file with new data, actually the file creation is based on the **jobDefineId** so it will create a new file each time that's why I was used it, but now I have changed it to CREATE to create new file each time either file is existing or not.
Thanks once again.